packages/zowe-explorer/src/uss/actions.ts

```diff
@@ -117,21 +117,6 @@ export async function refreshDirectory(node: IZoweUSSTreeNode, ussFileProvider:
     }
 }
 
-export async function createUSSNodeDialog(node: IZoweUSSTreeNode, ussFileProvider: IZoweTree<IZoweUSSTreeNode>): Promise<void> {
```

Comment:
Removed this because it's dead code that's not called anywhere and it's not a published API